app/models/zoom_meeting.rb

```diff
@@ -74,7 +75,14 @@ def create_participant_objects
   def uniq_participants_best_time(participants)
     grouped_by_name = participants.group_by(&:name)
     participants_best_time = grouped_by_name.map do |name, participant_records|
-      participant_records.min_by(&:join_time)
+      earliest_join_time = participant_records.min_by(&:join_time)
+      earliest_join_time.duration = calculate_duration(participant_records)
+      earliest_join_time
```

Comment:
I think this is a really good start! One potential issue we'll run into is that a student can have different names on a zoom call. Currently, we only connect one zoom alias for a student on a zoom call. That is the name they used that gives them the best join time. I'm thinking we will need to change that so students can have multiple zoom aliases on a call, and then we will need to sum up all of those durations as well.